node/README.md

```diff
@@ -1,62 +1,113 @@
+Initiates a new deployment to destroy and existing environment.
 
-`$ env0 --help`
+### `env0 approve <arguments>`
 
-### Options
+Runs an approval of an existing deployment that is waiting to be approved (AKA waiting in plan stage).
 
-**-k --apiKey** - Your env0 API Key
+### `env0 cancel <arguments>`
 
-**-s --apiSecret** - Your env0 API Secret
+Cancels an existing deployment that is waiting to be approved (AKA waiting in plan stage).
 
-**-o --organizationId** - Your env0 Organization id
+## Arguments Overview
 
-**-p --projectId** - Your env0 Project id
+The arguments are loaded from env0 configuration files, environment variables and CLI parameters.
+The order of precedence of the arguments is:
+1. `env0` CLI explicit parameters
+2. Environment Variables
+3. Configuration File
 
-**-b --blueprintId** - The Blueprint id you would like to deploy with (Optional for existing environments)
+### API Key
+> env0 API Key (Required)
+- Usage: `--apiKey`
+- Alias: `-k`
 
-**-e --environmentName** - The environment name you would like to create, if it exists it will deploy to that environment (Required for existing environments)
-  
- **-v --environmentVariables** - The environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"
+### API Secret
+> env0 API Secret (Required)
+- Usage: `--apiSecret`
+- Alias: `-s`
 
-**-q --sensitiveEnvironmentVariables** - Same as environment variables (`-v`) but will be marked as sensitive and hidden on env0 platform.
+### Organization ID
+> env0 Organization ID (Required)
+- Usage: `--organizationId`
+- Alias: `-o`
 
-**-r --revision** - You git revision, can be a branch tag or a commit hash. Default value `master`
+### Project ID
+> env0 Project ID (Required)
+- Usage: `--projectId`
+- Alias: `-p`
 
-**--archiveAfterDestroy** - Archive the environment after a successful destroy
+### Blueprint ID
+> The Blueprint ID you would like to deploy with (Required for new environments)
+- Usage: `--blueprintId`
+- Alias: `-b`
 
-**-h --help** - Get help
+### Environment Name
+> The environment name you would like to create, or deploy to an existing one (Required for existing environments)
+- Usage: `--environmentName`
+- Alias: `-e`
 
 ### Environment Variables
+> The environment variables to set on the deployed environment - works only on deploy and can be multiple, the format is "environmentVariableName1=value"
+- Usage: `--environmentVariables`
+- Alias: `-v`
+
+For sensitive environment variables, use:
+- Usage: `--sensitiveEnvironmentVariables`
+- Alias: `-q`
+
+### Revision
+> Your GIT revision, can be a branch, a tag or a commit hash. Defaults to `master`
+- Usage: `--revision`
+- Alias: `-r`
+
+### Archive After Destroy
+> Archives the environment after a successful destroy
+- Usage: `--archiveAfterDestroy`
+
+### Requires Approval
+> Require approval for deployment
+- Usage: `--archiveAfterDestroy`
```

Comment:
@yaronya  I bet that was a copy paste problem

Reply:
😅 
249b42e